docs/docs/language-support.mdx

```diff
@@ -12,9 +12,10 @@ from scratch with the data you provide.
 In addition, we also support pre-trained word embeddings such as spaCy. For information on
 what pipeline is best for your use case, check out [choosing a pipeline](./tuning-your-model.mdx#how-to-choose-a-pipeline).
 
-## Training a Model in Any Language
+## Training a Model in Any Languages
```

Comment:
Is this a typo or is it intentionally "any languages" now?

Reply:
We came to the conclusion that "many" sounded a bit strange. Therefore we figured just adding the "whitespace"-tokenizable part is "good enough for now". I'll likely do another iteration on this when the spaCy 3.0 stuff is merged. SpaCy will also start supporting Chinese, Japanese, Thai, Ukranian, Korean and Vietnamese.